### PR TITLE
Refs #9848: Move config file to config directory and provide via gem

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: csv.yml [hunk 1]
```diff
@@ -1,2 +0,0 @@
-:csv:
-  :enable_module: true
```
Comment (jlsherrill) [inline csv.yml]: isn't this file needed?

i chatted with tom and the other files in ./config/ really aren't needed at all.

Comment (ehelms) [inline csv.yml]: Weird, I swear I moved it instead of just removing it.

Comment (ehelms) [inline csv.yml]: Ah ha, .gitignore was ignoring all *.yml files

**Comment by ehelms** [comment]
Updated with your suggested removals

**Comment by jlsherrill** [comment]
APJ

**Pull request merged**